basic_integral/basic_integral.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+#include <cmath>
+using namespace std;
+
+int main() {
+	int N = 10, b, i;
+	double sum, fx;
+	double p = 2*acos(0.0);
+	
+	cout << "How many iterations?" ;
+	cin >> b ;
+	
+	N = 10 * pow(b-1,2);
+	
+	for(int i=0; i<N; i++){
+		fx = sin( (2*p*i/N) + p/N )*(2*p/N);
+		sum += fx;

```

Comment:
you have not initialized sum! So the first time this line of code is executed, C++ has to decide what the value sum is. Some compilers might take it as zero, which is good. But in general it gives you garbage data. This is a very dangerous thing in programming.